.github/workflows/run-unit-tests.yml

```diff
@@ -0,0 +1,32 @@
+# WARNING! It is currently being investigated how to make this faster
+# DO NOT blindly copy this workflow, not noticing the slow down,
+# because suddenly our tests will take hours to pass CI.
+# Hopefully this comment here will help prevent that.
+# https://github.com/MetaMask/metamask-extension/issues/25680
+
+name: Run unit tests
+
+on:
+  pull_request:
```

Comment:
We need to add a trigger for `develop` and `master` here as well. These long-running branches will not have PRs associated with them, but it's very important that we know that tests are passing on them.